containerWithMostWater.js

```diff
@@ -0,0 +1,23 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/container-with-most-water/
+
+const maxArea = (height) => {
+  let pointer1 = 0;
+  let pointer2 = height.length - 1;
```

Comment:
Typically it's bad practice to use `1, 2, ...` in var names, sometimes it's OK but here I feel like it would be better to call it `left` and `right` WDYT ?